src/components/Atlas/index.ts

```diff
@@ -0,0 +1,22 @@
+import { Coord, Layer } from "react-tile-map"
+import { Atlas } from "./Atlas"
+import {
+  AtlasColor,
+  AtlasInterface,
+  AtlasProps,
+  AtlasStateProps,
+  AtlasTileProps,
+  AtlasTileType,
+} from "./Atlas.types"
```

Comment:
This can be replaced by:
```suggestion
export * from "./Atlas.types"
```